kcidb/test_db.py

```diff
@@ -8,6 +8,7 @@
 import pytest
 import kcidb
 from kcidb.unittest import assert_executes
+from kcidb.db import Client
```

Comment:
Please remove this import, and refer to the `Client` class as `kcidb.db.Client`, like the rest of the code here does.